pennylane_lightning/src/Gates.cpp

```diff
@@ -220,6 +230,23 @@ Pennylane::RotationXGate::RotationXGate(double rotationAngle)
       js, c }
 {}
 
+const tuple<vector<CplxType>, double> Pennylane::RotationXGate::generator{
+    { 0, 1,
+      1, 0 }, -0.5
+};
+
+void Pennylane::RotationXGate::applyDerivative(const StateVector& state, const std::vector<size_t>& indices, const std::vector<size_t>& externalIndices) {
+    CplxType pref = (std::get<1>(Pennylane::RotationXGate::generator), 1.0);
```

Comment:
Why is it `(-0.5, 1)`?

Reply:
Oh, it's just me being confused. Should probably be written as 
```
1j * std::get<1>(Pennylane::RotationXGate::generator)
``` 
if I want to get `-0.5i`, right?